feature_engine/encoding/_docstrings.py

```diff
@@ -15,9 +15,15 @@
 
 _errors_docstring = """errors: string, default='ignore'
         Indicates what to do, when categories not present in the train set are
-        encountered during transform. If 'raise', then rare categories will raise an
-        error. If 'ignore', then rare categories will be set as NaN and a warning will
-        be raised instead.
+        encountered during transform. If 'raise', then unseen categories will raise an
+        error. If 'ignore', then unseen categories will be set as NaN and a warning will
+        be raised instead. If 'encode', then unseen categories will be encoded according
+        to the default strategy from the transformer, provided that it supports it.
+    """.rstrip()
+
+_errors_docstring_with_encode = _errors_docstring + """
```

Comment:
at the moment, only the count frequency has the "encode" extra functionality. And when we expand this to other classes, each one will do something different. So we would probably have to right independent docstrings for each one of them. I am not sure that having a centralized copy is suitable in this case. I would just re-word it in the class.

Reply:
But not all of the clases would end up supporting the "encode" option.